upgrade/scripts/upgrade/util/update-customizations.sh

```diff
@@ -162,7 +162,10 @@ if [ "$(yq4 eval '.spec.kubernetes.services.cray-sysmgmt-health.kube-prometheus-
   yq4 'del(.spec.kubernetes.services.cray-sysmgmt-health.victoria-metrics-k8s-stack.prometheus)' -i $c
   yq4 'del(.spec.kubernetes.services.cray-sysmgmt-health.victoria-metrics-k8s-stack.thanos)' -i $c
 fi
-
+#sma-vm-cluster
+if [ "$(yq4 eval '.spec.kubernetes.services.sma-vm-cluster' $c)" == null ]; then
+  yq4 -i eval ".spec.kubernetes.services[\"sma-vm-cluster\"] += (load(\"${upgrade_customizations}\") | .spec.kubernetes.services[\"sma-vm-cluster\"])" "$c"
```

Comment:
Since CSM 1.6.1 is a patch release, this code must work with both CSM 1.6.0 and 1.6.1. I think CSM 1.6.0 will come with [customizations file](https://github.com/Cray-HPE/csm/blob/release/1.6.0/vendor/github.com/Cray-HPE/shasta-cfg/customizations.yaml), where `sma-vm-cluster`section is absent, so this statement will fail. The `sma-vm-cluster` section was only added in [CSM 1.6.1 customizations file](https://github.com/Cray-HPE/csm/blob/release/1.6/vendor/github.com/Cray-HPE/shasta-cfg/customizations.yaml#L675).

Reply:
done the changes